.migration-backup/ui-components/app/(management)/setup.tsx

```
@@ -0,0 +1,188 @@
+import React, { useState } from 'react';
+import {
+  Text,
+  TextInput,
+  TouchableOpacity,
+  StyleSheet,
+  Alert,
+  ScrollView,
+  ActivityIndicator,
+  SafeAreaView
+} from 'react-native';
```

ESLint

[error] There should be at least one empty line between import groups

(import/order)